### PR TITLE
add Liked page and Toast component

### PR DESCRIPTION
**Pull request by hsnice16**
### This PR will add code for the Liked page (Design and Functionality) and I have also added the Toast component

In this PR, **476 additions and 52 deletions** lines of code
- if you don't like you can avoid reviewing Backend related files
- some files contain only 5 to 6 lines of code

### Please review this PR and give your feedback
---
to Log In, use the following credentials
```
email: johndoe@gmail.com
password: johnDoe123
```
---
#### [Live URL](https://dekho-video-git-page-liked-hsnice16.vercel.app/)
image

### CODE REVIEW

File: src/components/compose/Compose.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
```
Comment (mikesch803) [inline src/components/compose/Compose.jsx:1]: i dont understand why you have this component

**Review by mikesch803 — APPROVED** [review]
Your code is clean and well structured, one thing as per prd you have add like/dislike btn to video-listing/home page

File: src/components/compose/Compose.jsx (hunk 1 above)
Comment (hsnice16) [reply src/components/compose/Compose.jsx:1]: this is a package. it helps to define the types of props.

**Review by Developers-codz — APPROVED** [review]
Well Structure code and folder. Went through all the code unable to find any mistakes. Everything looks good to merge

File: src/components/compose/Compose.jsx (hunk 1 above)
Comment (mikesch803) [reply src/components/compose/Compose.jsx:1]: i m not saying about prop types but this compose component

Comment (hsnice16) [reply src/components/compose/Compose.jsx:1]: oh! sorry.
the `Compose` component, I have used this to compose all the context providers in the `index.js` file of the `src`.
As the providers number was getting increase and the file was getting messy, that's why I created this component.

**Comment by hsnice16** [comment]
Thank you for reviewing,

File: src/components/compose/Compose.jsx (hunk 1 above)
Comment (mikesch803) [reply src/components/compose/Compose.jsx:1]: Nice! that thing is new to me, it actually looks clean

Comment (hsnice16) [reply src/components/compose/Compose.jsx:1]: Yeah, I also search for it when it started to get messy.

**Pull request merged**